script/chart-museum.sh

```diff
@@ -88,17 +79,47 @@ installChartMuseum() {
     --set env.secret.BASIC_AUTH_PASS=$CHARTMUSEUM_PWD
   info "Waiting for ChartMuseum to be ready..."
   kubectl rollout status -w deployment/chartmuseum --namespace=${CHARTMUSEUM_NS}
+  
+  echo "Installing Ingress for ChartMuseum with access through host ${CHARTMUSEUM_HOSTNAME}"
+  kubectl create -n $CHARTMUSEUM_NS -f - -o yaml << EOF
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    nginx.ingress.kubernetes.io/connection-proxy-header: keep-alive
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "600"
+    nginx.ingress.kubernetes.io/proxy-buffer-size: "8k"
+    nginx.ingress.kubernetes.io/proxy-buffers: "4.0"
```

Comment:
not sure we really need these annotations here, no?

Reply:
We are using those annotations in the other Ingress definition we use, so I just copy-pasted. Happy to remove them.